packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -56,6 +41,10 @@ const propTypes = {
    * If set to true, the modal dialog with have overflow-y set to scroll.
    */
   isScrollable: PropTypes.bool,
+  /**
+   * Ref callback for abstract modal.
+   */
+  refCallback: PropTypes.func,
```

Comment:
If you did use createRef above, you'd have to define this proptype as:
`PropTypes.shape({ current: PropTypes.instanceOf(Element) })`

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/608/commits/148cfbfb7fa7abcf5cd1ae14bf89bf2be7496b9d